extensions/amp-story/1.0/background.js

```diff
@@ -123,14 +123,20 @@ export class AmpStoryBackground {
 
     setStyle(hidden, 'background-image', 'none');
 
+    // Set css background properties immediately so it will always be overridden
+    // by the image if there is one.
+    setStyle(hidden, 'background', cssBackground);
+
     // Image will be swapped on load.
     whenFresh(imgLoad, () => {
-      setStyle(hidden, 'background-image', url ? `url(${url})` : null);
+      if (!url) {
```

Comment:
Should we move the `setStyle(hidden, 'background', cssBackground);` above to here so that we avoid doing two paints when there is an image? Or maybe it's not the best idea since we have to wait for the image to load to set a color.

Reply:
☹️ yeah, that's why I did it the way I did.    Not sure if there's a better way.